src/ngraph/runtime/gpu/gpu_util.cpp

```diff
@@ -77,3 +77,101 @@ void runtime::gpu::cuda_memset(void* dst, int value, size_t buffer_size)
 {
     cudaMemset(dst, value, buffer_size);
 }
+
+namespace
+{
+    uint64_t _powU64(uint64_t base, uint64_t exp)
```

Comment:
any name starting with an underscore is reserved in c++ so don't start functions or variable names with an underscore.
https://stackoverflow.com/questions/228783/what-are-the-rules-about-using-an-underscore-in-a-c-identifier

Reply:
Yeah, I originally put them in an anonymous namespace for that reason but I've removed the prefix just in case.